CONTRIBUTING.md

```diff
@@ -17,10 +17,18 @@ This will create an isolated Workspace in vscode, including all tools required t
 
 Now you can compile and install the extension
 ```bash
+git submodule update --init --recursive
 make debug
 make install
 ```
-then connect to Postgres using `psql`.
+Then, connect to PostgreSQL using psql:
+```bash
+psql -U postgres -d mooncake
+```
```

Comment:
PostgresSQL -> Postgres for consistency
I think `psql` should just work if you are using dev container, no need for `-U postgres -d mooncake`